app/views/container_replicator/show.html.haml

```diff
@@ -1,4 +1,6 @@
 - if @display == 'containers'
```

Comment:
Can't you just do:

    - if ['containers', 'container_groups'].includes?(@display)
      = render :partial => "layouts/gtl", :locals => {:action_url => "show/#{@record.id}"}

(there should be other places where we do something similar)